### DISCUSSION TITLE
FEATURE REQUEST: Bladebit support

### DISCUSSION THREAD

**Discussion by jonathanhatch** [swar/Swar-Chia-Plot-Manager#1275] [Ideas]
This is something I'd love to see. The specific issue I'm having is lining up a buffer drive to push the plots into so I have less time between plots, so while bladebit is back to plotting, the previous plot is being pulled from, say, a raid-0 array of enterprise sas3 SSDs and put into destination HDDs.

Is this possible?

**Comment by robkthx** [comment]
I would also like to see support for bladebit
